suite/tests/client-interface/ldstex.c

```diff
@@ -843,6 +881,64 @@ GLOBAL_LABEL(FUNCNAME:)
         END_FUNC(FUNCNAME)
 #undef FUNCNAME
 
+#ifdef AARCH64
+/* int ldstex_inc32_with_xzr(two_counters_t *counter)
+ */
```

Comment:
These don't test the case where the exclusive load and store are in different blocks. Do we test that path using the clean-calls case below? (which thwarts single-block optimizations for ldstex2cas)

Reply:
Yes -- added a comment.  The slowpath is the same regardless of whether actually in separate blocks.